### ISSUE TITLE
Persist on reboot

### ISSUE THREAD

**Issue by notenti** [dylanaraps/wal#10]
I've attempted a few different configurations to have the theme persist on a boot, but with no luck. Currently, my `~/.zsh` has `(wal -r &)` in it, my `.xinitrc` has `wal -i "$(< "${HOME}/.cache/wal/wal")"` in it, and my `i3 config` has `exec wal -i $HOME/Pictures/rowboat.jpg` in it. Is there a way to have the theme still be applied on boot?

**Comment by dylanaraps** [comment]
Here's how my system is setup (Theme persists on boot):

**.bashrc** / **.zshrc**

```sh
(wal -r &)
```

**i3 Config**

```sh
exec wal -i "$(< "${HOME}/.cache/wal/wal")" 
```

I don't have any `wal` related lines in my `.xinitrc`. Try mimicking this configuration and seeing it if works.

**Comment by notenti** [comment]
Hmm still nothing. As soon as I execute `wal -i ~/Pictures/rowboat.jpg` it works fine. Also, I can execute `wal -i "$(< "${HOME}/.cache/wal/wal")" ` in my terminal and it works fine then, too. Must be something in the i3 config.

**Comment by notenti** [comment]
Actually, I stand corrected. I **cannot** simply execute `wal -i "$(< "${HOME}/.cache/wal/wal")"` after a boot and see the colors change.

**Comment by dylanaraps** [comment]
Does the file `${HOME}/.cache/wal/wal` exist after a reboot? What that command is doing is getting the current wallpaper from the file contents of that file.

**Comment by notenti** [comment]
Yeah, it's there along with `colors/`, `colors.scss`, `colors.sh`, & `schemes`.

**Comment by dylanaraps** [comment]
It might have something to do with `zsh` maybe? Does `zsh` support this syntax?: `$(< file)"`

**Comment by dylanaraps** [comment]
Also, can you send me your `i3` config? I'll try and reproduce the issue using it.

**Comment by notenti** [comment]
After some more trial and error, executing `wal -i "$(< /home/nate/.cache/wal/wal )"` does work every time. Something must have happened early causing it to break. I attached my `i3 config` file (I made it a .txt so github would let me upload it). 
[i3config.txt](https://github.com/dylanaraps/wal/files/729396/i3config.txt)
The command is way at the bottom. I have tried many different variations of the quotes, but nothing has worked yet. I've also put the command in a script and called `exec ~/script.sh` in my `i3 config`, but that didn't work either.
